Update README.md

rxy94 opened the pull request:
Subida del enlace al repositorio Pixela de Ruyi y Pablo

rxy94 committed:
Update README.md

Subida del enlace al repositorio Pixela de Ruyi y Pablo

DavidHormigoRamirez reviewed (changes requested):
1. Poned los dos alumnos apuntando el mismo proyecto: una sola linea por proyecto, luego todos los alumnos separados por <br> que participen
2. Cambiad el enlace para que refleje el nombre de vuestro proyecto: en lugar de proyecto que ponga Pixela

rxy94 committed:
Update README.md

- Sustituyendo el link Proyecto por el nombre del proyecto - Pixela
- Poniendo a los participantes del proyecto en la misma fila para que apunten al mismo proyecto

rxy94 committed:
Update README.md

rxy94 committed:
Update README.md
